src/library-authoring/common/specs/helpers.jsx

```diff
@@ -1,12 +1,20 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-import jest from 'jest-mock';
 import { render } from '@testing-library/react';
-import AppContext from '@edx/frontend-platform/react/AppContext';
 import { IntlProvider } from '@edx/frontend-platform/i18n';
 import * as React from 'react';
-import { Provider } from 'react-redux';
 import { getDefaultMiddleware } from '@reduxjs/toolkit';
+import { initialize } from '@edx/frontend-platform';
+import AppContext from '@edx/frontend-platform/react/AppContext';
+import { Provider } from 'react-redux';
+import { createBrowserHistory } from 'history';
```

Comment:
[frontend-platform exposes a history object](https://edx.github.io/frontend-platform/module-Initialization.html#~history) which you can get with `import { history } from '@edx/frontend-platform'`. Would that work here?

Reply:
I don't think so. That history object is made in the same way we're doing here, but if we used it, it would be the same object between tests, which risks polluting the environment. We're injecting the newly created history in the same way the upstream platform library does, so this shouldn't have ill effects.